tests/dialects/test_clickhouse.py

```diff
@@ -369,6 +369,7 @@ def test_clickhouse(self):
             "SELECT STARTS_WITH('a', 'b'), STARTSWITH('a', 'b')",
             write={"clickhouse": "SELECT startsWith('a', 'b'), startsWith('a', 'b')"},
         )
+        self.validate_identity("SYSTEM STOP MERGES foo.bar")
```

Comment:
This silences the relevant warning when running the test suite.
```suggestion
        self.validate_identity("SYSTEM STOP MERGES foo.bar", check_command_warning=True)
```

Reply:
Will keep that in mind for my future contributions, thanks!